pep-9999.rst

```diff
@@ -0,0 +1,65 @@
+PEP: 9999
+Title: Persistent cache for packaging frontends
+Author: Bernát Gábor <gaborjbernat@gmail.com>
+Sponsor: TBD
+Discussions-To: https://discuss.python.org/t/proposal-adding-a-persistent-cache-directory-to-pep-517-hooks/2303/57
```

Comment:
Would it be better to start a new thread instead? This thread is already quite long and can be difficult to follow for newcomers.